### ISSUE TITLE
Context menu: aria attributes not on tab role element (4.1.2)

### ISSUE THREAD

**Issue by Blackbaud-MattGregg** [blackbaud/skyux1#840]
### Expected behavior

For an accordion, the tab role DOM element has the aria-expanded and aria-controls attributes. Being properties of the tab element so they can be accessed by assistive technology as such.
### Actual behavior

These two aria attributes are on a div within the tab element instead of being on the element with the tab role. 
### Steps to reproduce
1. Go to 3rd context menu example http://skyux.developer.blackbaud.com/components/contextmenu/
2. Open context menu and review accordion menu.

**Comment by Blackbaud-PatrickOFriel** [comment]
Hey @Blackbaud-MattGregg I was looking at aria-expanded here https://www.w3.org/TR/wai-aria/states_and_properties#aria-expanded and it doesn't say to use this with the tab role (though it does say that it "inherits" into the tab role, but I don't really know what that means). I see that the aria-selected attribute seems to be used with the tab role instead, so should I just use that instead of aria-expanded, or use both aria-expanded and aria-selected, or use only aria-expanded?

**Comment by Blackbaud-PatrickOFriel** [comment]
Though looking at it again right now what I'm thinking is this (and correct me if I'm wrong right here). Have the role tab with aria-selected surrounding the div with role button and the aria expanded and controls. Going from:

```
<div role="tab" id="{{::headingId}}" aria-selected="{{isOpen}}"  class="panel-heading" tabindex="0" ng-keypress="toggleOpen()">
    <h4 class="panel-title">
      <div aria-expanded="{{isOpen}}" aria-controls="{{::panelId}}" data-toggle="collapse" class="accordion-toggle" uib-accordion-transclude="heading"><span uib-accordion-header ng-class="{'text-muted': isDisabled}">{{heading}}</span></a>
    </h4>
  </div>
```

to: 

```
<div role="tab" id="{{::headingId}}" aria-selected="{{isOpen}}"  class="panel-heading" tabindex="0" ng-keypress="toggleOpen()">
    <h4 class="panel-title">
      <a role="button" aria-expanded="{{isOpen}}" aria-controls="{{::panelId}}" data-toggle="collapse" class="accordion-toggle" uib-accordion-transclude="heading"><span uib-accordion-header ng-class="{'text-muted': isDisabled}">{{heading}}</span></a>
    </h4>
  </div>
```

(I also noticed that there was another error in the original template where the div with the aria-controls and expanded was closed by an anchor tag instead of a div. I think it was originally intended to be an anchor with the role button)

**Comment by Blackbaud-MattGregg** [comment]
The aria-expanded property can be used with either element. In the case of the accordion, the only part of the authoring criteria this update might not hit is having the tab element with that expanded state relationship - https://www.w3.org/TR/wai-aria-practices-1.1/#accordion  -"An accordion should manage the expanded/collapsed state of each tab by maintain its aria-expanded state."

The other consideration here is just simplicity. I'm not sure if the h4 and button are needed for some other purpose(s), but seems like this could be simplified to just have a the tab and instead of a button inside of a tab which someone would have to listen to each time.

**Comment by Blackbaud-PatrickOFriel** [comment]
Gotcha, I'll simplify it then and see if I can put all aria-related attributes on tab without affecting it in any other way

**Comment by Blackbaud-PatrickOFriel** [comment]
In [v1.7.6](https://github.com/blackbaud/skyux/releases/tag/v1.7.6)
